components/profile_popover/profile_popover.tsx

```diff
@@ -617,31 +619,50 @@ class ProfilePopover extends React.PureComponent<ProfilePopoverProps, ProfilePop
             </OverlayTrigger>
         );
 
-        const startCallMessage = formatMessage({
-            id: t('webapp.mattermost.feature.start_call'),
-            defaultMessage: 'Start Call',
-        });
-        const callButton = (
-            <OverlayTrigger
-                delayShow={Constants.OVERLAY_TIME_DELAY}
-                placement='top'
-                overlay={
-                    <Tooltip id='startCallTooltip'>
-                        {startCallMessage}
-                    </Tooltip>
-                }
-            >
-                <button
-                    type='button'
-                    className='btn icon-btn'
+        const renderCallButton = () => {
+            if (!this.props.isCallsEnabled) {
+                return null;
+            }
+            const disabled = this.props.isUserInCall || this.props.isCurrentUserInCall;
+            const startCallMessage = this.props.isUserInCall ? formatMessage({
+                id: t('user_profile.call.userBusy'),
+                defaultMessage: 'User is in another call',
```

Comment:
@matthewbirtch this message is shown to user in tooltip, when the user that current user trying to call is in another call. Please feel free to suggest different wording.

Reply:
Nice! I like this

Reply:
Thanks, done